app/controllers/api/v1/users/books_controller.rb

```diff
@@ -47,16 +50,20 @@ def update
       @user_book.update(current_page: params[:current_page])
       render json: @user_book
     else
-      render json: {
-        error: "Could not find record with " \
-               "user_id: #{params[:user_id]}, " \
-               "book_id: #{params[:id]}"
-      }, status: 404
+      return cannot_find_user_book(params)
     end
   end
 
-  private
+  ## move to application controller?
+  def cannot_find_user_book(params)
```

Comment:
I like this. Nice refactor

Reply:
Updated. #cannot_find_user_book moved private method.